src/main/resources/assets/carpet/scripts/draw_beta.sc

```diff
@@ -15,7 +15,14 @@ __config() -> {
         'cuboid <center> <radius> <height> <orientation> <block> <hollow>'->['draw_prism', null, true],
         'cuboid <center> <radius> <height> <orientation> <block> <hollow> replace <replacement>'->['draw_prism', true],
         'cylinder <center> <radius> <height> <orientation> <block> <hollow>'->['draw_prism', null, false],
-        'cylinder <center> <radius> <height> <orientation> <block> <hollow> replace <replacement>'->['draw_prism', false]
+        'cylinder <center> <radius> <height> <orientation> <block> <hollow> replace <replacement>'->['draw_prism', false],
+        'cache <mode>'->'set_cache_mode',
```

Comment:
not sure why you need a cache. setting the block should be majority of the resources it should require.

Reply:
Nah man, it rly helps. (at least it does on my craptop)

Reply:
that's because the sphere drawing code is wrong.

Reply:
k then, what code should I use?